sechub-web-ui-solution/helm/sechub-web-ui/values.yaml

```
@@ -0,0 +1,76 @@
+# SPDX-License-Identifier: MIT
+
+# This is a sample values file containing the defaults.
+
+name: web-ui
+
+image: "ghcr.io/mercedes-benz/sechub/sechub-web-ui:latest"
```

We should make the image pull policy configurable (did you merge the `develop` branch?)
```suggestion
image: "ghcr.io/mercedes-benz/sechub/sechub-web-ui:latest"
imagePullPolicy: "IfNotPresent"
```